lib/include.php

```diff
@@ -10,17 +10,20 @@ function __autoload($class) {
 }
 
 session_start();
+require_once 'Pheanstalk/ClassLoader.php';

```

Comment:
Yes, you have. 

See:
https://github.com/ptrofimov/beanstalk_console/blob/master/lib/Pheanstalk/ClassLoader.php